package.json

```diff
@@ -16,7 +16,7 @@
     "build": "tsc --build --verbose ./tsconfig.build.json",
     "lint": "eslint --fix src",
     "style": "prettier --write ./src/**/*.ts",
-    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit",
+    "test": "mocha -r ts-node/register 'test/**/e2e.test.ts' --exit --timeout 150000",
```

Comment:
nit: fine for now to get these changes but make sure to change this back, you can set a long timeout in a mocharc config file

Reply:
should have removed this actually, will do now